### DISCUSSION TITLE
Sound Play by Distance

### DISCUSSION THREAD

**Discussion by ghost** [mozilla/hubs#5555] [General Questions & Discussion]
Places the sound near a specific object and plays the sound when approaching. The sound disappears when you leave. Is it feasible to play the sound from the beginning when entering the prefecture for a certain distance?

I understand the audio distance attenuation function, but I would like to know how to play the audio from the beginning when it goes inside a certain distance.

**Comment by pattersonbl2** [comment]
We have not run into setup yet.  There may be someone in the discord that could help with this.
